src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1662,4 +1662,37 @@
   <data name="MakeParameterlessConstructorPublic" xml:space="preserve">
     <value>Make the constructor that takes zero parameters 'public'</value>
   </data>
+  <data name="DynamicInterfaceCastableImplementationUnsupportedDescription" xml:space="preserve">
+    <value>Providing a functional 'DynamicInterfaceCastableImplementationAttribute'-attributed interface requires the Default Interface Members feature, which is unsupported in Visual Basic.</value>
+  </data>
+  <data name="DynamicInterfaceCastableImplementationUnsupportedMessage" xml:space="preserve">
+    <value>Providing a 'DynamicInterfaceCastableImplementation' interface in Visual Basic is unsupported</value>
+  </data>
+  <data name="DynamicInterfaceCastableImplementationUnsupportedTitle" xml:space="preserve">
+    <value>Providing a 'DynamicInterfaceCastableImplementation' interface in Visual Basic is unsupported</value>
+  </data>
+  <data name="ImplementInterfacesOnDynamicCastableImplementation" xml:space="preserve">
+    <value>Implement inherited interfaces</value>
+  </data>
+  <data name="InterfaceMethodsMissingImplementationDescription" xml:space="preserve">
+    <value>Types attributed with 'DynamicInterfaceCastableImplementationAttribute' act as an interface implementation for a type that implements the 'IDynamicInterfaceCastable' type. As a result, it must provide an implementation of all of the methods defined in the inherited interfaces, because the type that implements 'IDynamicInterfaceCastable' will not provide them otherwise.</value>
```

Comment:
Is `IDynamicInterfaceCastable` is a must to use `DynamicInterfaceCastableImplementationAttribute`? Any separate analyzer work for that?

Reply:
`IDynamicInterfaceCastable` is a must-use type related to the `DynamicInterfaceCastableImplementationAttribute` type. But since `IDynamicInterfaceCastable` will be implemented by a different type, so there's no analyzer work for that.